src/components/Inputs/BirthdateInput/__snapshots__/BirthdateInput.test.js.snap

```diff
@@ -37,6 +38,7 @@ Array [
     }
     name="birthdate-auto-corrected"
     onBlur={[Function]}
+    pattern="[0-9]*"
```

Comment:
Can you remind me what's the '*' doing on this particular pattern?

Reply:
`[0-9]*` should match any `n` amount of characters contained in the the `[0=9]` set, where `n >= 0`

Reply:
Ok, so it's a literal regex metacharacter thx for confirming @bwalker8038

Reply:
I think not having this means the regular keyboard is open, but with the modifier that opens the number keys on the top row (instead of the 3x3 pad we want).